packages/Syntax/src/globalStyles.tsx

```diff
@@ -0,0 +1,121 @@
+import { Variant } from './types';
+import { injectGlobal } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+
+interface Base16Palette {
+  '00': string; // Background
```

Comment:
So – originally these were [base16](https://github.com/chriskempson/base16)-formatted palettes. Turns out it's MUCH easier to make a color palette that's up to our accessibility guidelines with this set of 11. So partly, the numbering is legacy.

The reason to keep these numbers came from the desire for flexibility in the future. If we named the colors after their use, it becomes much harder to change how the colors are used.

Additionally, naming the colors after one of their uses falsely implies that's the only use. In the comments below, we have some of these colors covering a wide variety of programming features, etc.

I needed to put the colors in some sort of syntax-highlighting specific color framework. If you have a good naming convention that might work better, let me know!

Reply:
Yeah that makes sense.

If anything I'd just make this an array so when referencing values you can do a clean lookup via a number and not via a strong. Then you can also have the array be `as const` so it's immutable as well.

Reply:
The reason not to do this is because it's harder to find the correct index of the desired element when looking at this.

If it's just
```
const myPalette = [
    uiColors.gray.light3,
    uiColors.gray.light2,
    uiColors.gray.dark2,
    uiColors.white,
];
```
it becomes substantially harder to figure out how to reference the correct color.

Reply:
Fair enough. Can we make these numbers then? so `{ 0: string }` etc.? Strings are hard to minify.